Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.35244&attempt_id=6c0c8534-ec86-47e5-8015-295b1c0c66e8)

<details><summary>Partial log (click to expand)</summary><p>

```
<       ld      e,l
---
>       ld      a,e
>       and     18
>       ld      e,a
make[1]: *** [Makefile:28: Issue_510_long_and.opt] Error 1
make[1]: Leaving directory '/build/source/testsuite'
make: *** [Makefile:135: testsuite] Error 2
builder for '/nix/store/k8i83lh634n6yjpfx5aqls6y8bk5w07h-z88dk-20180217.drv' failed with exit code 2
[31;1merror:[0m build of '/nix/store/k8i83lh634n6yjpfx5aqls6y8bk5w07h-z88dk-20180217.drv' failed
```
</p></details>

--- a/pkgs/development/compilers/z88dk/default.nix
+++ b/pkgs/development/compilers/z88dk/default.nix
@@ -45,6 +45,6 @@ stdenv.mkDerivation rec {
     description = "z80 Development Kit";
     license     = licenses.clArtistic;
     maintainers = [ maintainers.genesis ];
-    platforms = platforms.linux;
+    platforms = [ "x86_64-linux" ];
   };
 }

